packages/site/src/layout/ComponentView.tsx

```diff
@@ -49,19 +49,23 @@ export const ComponentView = () => {
                     </CodePreviewWindow>
                   )}
                 </Box>
-                <Tabs onTabChange={updateStyles}>
-                  <Tab label="Design">
-                    <Content spacing="large">
-                      <ComponentContent />
-                    </Content>
-                  </Tab>
-                  <Tab label="Props">
-                    <PropsList values={stateValues} />
-                  </Tab>
-                  <Tab label="Code">
-                    <CodeViewer code={code} />
-                  </Tab>
-                </Tabs>
+                <span
+                  style={{ "--public-tab--inset": 0 } as React.CSSProperties}
```

Comment:
Properly aligns the Tabs and doc content to the rest of the layout
![image](https://github.com/user-attachments/assets/f798080e-6cd5-4b06-84b7-04f0deeb31cc)

Reply:
Is this something we should ship in Tabs??

Reply:
I think this implementation exposes a potential need for it - right now that `public` inset is used so that in cases like Modals where we "automatically" handle alignment for consumers, it Just Works as the Modal's padding increases/decreases across breakpoints
<img width="633" alt="image" src="https://github.com/user-attachments/assets/aeaafa58-c865-4cc3-8952-8485d493f0af">

But this also feels like maybe Tabs shouldn't have an opinion about how it's aligned, and that should be left to the `<Content>` or `<Flex>` or what-have-you that's wrapping it